app/contributors/[slug]/page.tsx

```diff
@@ -346,7 +346,7 @@ export default async function Contributor({ params }: Params) {
         {contributor["activityData"] &&
           contributor["activityData"]["activity"] && (
             <div className="mt-6 px-4 md:p-0">
-              <div className="flex flex-col gap-1">
+              <div className="flex flex-col gap-1 font-inter">
```

Comment:
Let's keep the default `font-mono` itself across the entire leaderboard. Making it different in some parts and different in other parts doesn't give a consistent design.

```suggestion
              <div className="flex flex-col gap-1>
```